lornatumuhairwe/tofaanga
# Flask setup is completed

**what does this PR do?**
This is the final folder setup of the project before the OOP is implemeted

**where should the reviewer start?**
Check the app folder to see the Flask setup files

**How should this be manually tested?**
Write python run.py in terminal and press enter. The flask server should run at that point.

**Any background context you want to provide?**
The project had a different folder setup that the product requirements. This PR is a final setup from challenge 1. and also includes the flask setup. This will provide a starting point to the OOP section of the code.

What are the relevant tickets?
No

Questions?
Is there a blog post? N
Does the knowledge base need an update? N
@emugaya @Thegaijin @skipcheru @andela-gochieng
